sharedbuffers/pool.py

```diff
@@ -0,0 +1,209 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
```

Comment:
Aren't you going to cythinize this file?

Reply:
I don't see the need for it ATM